code/lambda/getsingulartooldata/src/main.rs

```diff
@@ -1,97 +1,68 @@
 use aws_sdk_dynamodb::types::AttributeValue;
```

Comment:
If this is unused, id like to see a top level comment here saying that this code doesn't run and to check the python code out. Plus a description of why doing this in rust was hard and where you got stuck so that someone reviving this later can get a jump start into your thought process

Reply:
Donee see [b3c0fae](https://github.com/illusion173/CyberToolAnalysis/pull/42/commits/b3c0fae8c549a930c22d62e4f33267777e5b8db3)